packages/@cdktf/cli-core/src/lib/models/deploy-machine.ts

```diff
@@ -232,7 +232,7 @@ export const deployMachine = createMachine<
               OVERRIDE: {
                 target: "processing",
                 actions: send(
-                  { type: "SEND_INPUT", input: "override\r\n" },
+                  { type: "SEND_INPUT", input: "override\n" },
```

Comment:
I'm not entirely sure, but I think this came in as a fix for Windows (at least for the approval workflow) – so maybe it's required for overrides for Windows, too?
Do you remember whether that was required for Windows, @DanielMSchmidt?

Reply:
Yeah windows does not take the lines without the `\r`. Maybe we need a helper to make this inputs so we don't forget about it from here on out

Reply:
I like that idea!

Reply:
So, this one is a bit tricky, and I think the `\r` fix has to be applied carefully.

I spent quite a bit of time after I added this because it broke the approval flow right after the override flow, and I couldn't figure out why 😓 . Apparently `\r` gets accepted as a newline (because I think older mac OSes [used it as a newline](https://stackoverflow.com/a/3720674)) and then the following `\n` gets accepted as a legit response to `Approve?` which is not what we want. I expect this to be a problem with our multi-stack deployments too on mac now.

I have a follow up item to see how we can detect the OS and then add an `\r` if it's windows only.